src/html5_parser/node.rs

```diff
@@ -398,7 +398,7 @@ mod test {
     use super::*;
 
     #[test]
-    fn test_new_document() {
+    fn new_document() {
```

Comment:
In Rust, it's fine to avoid the `test_` prefix (probably because we're already in a `tests` module).